scanners/kubeaudit/templates/kubeaudit-scan-type.yaml

```diff
@@ -8,7 +8,12 @@ spec:
     location: "/home/securecodebox/kubeaudit.jsonl"
   jobTemplate:
     spec:
-      ttlSecondsAfterFinished: 10
```

Comment:
Is this an important default we should define?

Reply:
Mh no this seems to have been forgotten by the ttl pull request.
Good catch